builder-base/install.sh

```diff
@@ -62,6 +64,18 @@ tar -C /usr/local -xzf go${GOLANG_VERSION}.linux-amd64.tar.gz
 rm go${GOLANG_VERSION}.linux-amd64.tar.gz
 mv /usr/local/go/bin/* /usr/bin/
 
+# install amazon-ecr-credential-helper
+# We are installing a specific commit of this because we need the sts regional endpoint changes in the aws sdk
+# to avoid hitting the global sts endpoint.
+# Commit: https://github.com/awslabs/amazon-ecr-credential-helper/commit/a004738dbac968cb287b47ae8ca39fd3b451e547
```

Comment:
Can we update this comment to mention tagged release instead because we changed it below?

Reply:
There is actually a v0.5.0 release in amazonlinux now, switching to install that via yum.

```
bash-4.3# yum info amazon-ecr-credential-helper  
Loaded plugins: ovl, priorities
Available Packages
Name        : amazon-ecr-credential-helper
Arch        : x86_64
Version     : 0.5.0
Release     : 1.amzn2
Size        : 2.3 M
Repo        : amzn2extra-docker/2/x86_64
Summary     : Amazon ECR Docker Credential Helper
License     : Apache 2.0
Description : The Amazon ECR Docker Credential Helper is a credential helper for the Docker
            : daemon that makes it easier to use Amazon Elastic Container Registry.

```

https://github.com/awslabs/amazon-ecr-credential-helper/releases/tag/amzn2%2F0.5.0-1